### ISSUE TITLE
Differentiate tabs with different requests, but matching titles

### ISSUE THREAD

**Issue by CeterusMike** [a85/POSTMan-Chrome-Extension#8954]
**Is your feature request related to a problem? Please describe.**
When working with generically-named requests across two different collections or folders within the same collection, there isn't a quick way to differentiate open tabs for each.

For example, I'm currently working on a system where we have routes like this:

- /api/accounts/service1/data
- /api/accounts/service2/data

Each of these is titled "Get Account Data" in the Postman collection, but they are stored in different folders within the collection.

> Collection
>> Service1
>>> Get Account Data

>> Service2
>>> Get Account Data

If I have tabs for each open at the same time, it's impossible to tell which is which without clicking into the tab and examining the route. I realize renaming these is probably the safest way to go but if this is an external collection that came with a lot of requests that may not be feasible.

**Describe the solution you'd like**
If multiple tabs have identical names, append the folder name (or collection name, whichever is the closest that is unique).

- If request titles are the same, get the folder for each and show **Get Account Data (Service1)**.
- If for some reason the folders are the same but they are in different collections (a v1 vs v2 collection, perhaps?), you could instead use the collection name: **Get Account Data (v1)**.

**Describe alternatives you've considered**
Just any way to tell them apart, really. Color-code them or something, I honestly do not have a preference as long as I can stop clicking into each tab and checking the route or memorizing the order. 😬

**Comment by vvs11** [comment]
@CeterusMike Thanks for writing in! This is an issue raised by other users as well and we are working on finding a fix for this. Meanwhile, you can go through #8486. There's a workaround presented there. Let us know if it unblocks you for now.

**Comment by CeterusMike** [comment]
@vvs11 The sidebar movement feature is helpful, but it does require clicking the tab in order to figure out which is which. That ends up being the same process as what I mentioned, where I have to click into the tab and look at the endpoint. The only real difference is which part of the screen I look at after the click. I was hoping to be able to tell the tabs apart before clicking one.

I do see now that #8629 appears to be very similar to what I've requested (except that one is a request for a rollover function rather than showing in the tab name all the time), so if you guys want to close my request out as a duplicate of that one, that is fine with me!

**Comment by vvs11** [comment]
Thanks for the input @CeterusMike I will mark this as duplicate and let's keep the conversation going on the other issue 

Duplicate of #8629
